lib/cryptoexchange/exchanges/coss/services/pairs.rb

```diff
@@ -10,8 +10,8 @@ def fetch
         end
 
         def adapt(output)
-          output['result'].map do |value|
-            base, target = value['MarketName'].split('-')
+          output.map do |(market_name, value)|
```

Comment:
same here, no brackets required. `_value` for unused variable.

Reply:
👍